hr_employee_calendar_planning/migrations/13.0.1.3.0/post-migration.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if version >= "13.0.1.0.0":
+        calendars_two_weeks = env["resource.calendar"].search(
+            [("auto_generate", "=", False), ("two_weeks_calendar", "=", True)]
+        )
+        calendars = env["resource.calendar"].search(
+            [("auto_generate", "=", False), ("two_weeks_calendar", "=", False)]
+        )
+        for employee in env["hr.employee"].search([]):
+            if bool(
+                employee.mapped("calendar_ids.calendar_id").filteed(
```

Comment:
You have to check if there's any employee with mix of two and one calendar to regenerate. The rest are OK

Reply:
In my opinion, all the employees that in their auto-generated calendar have a calendar of 2 weeks and another without 2 weeks should be regenerated because so far the auto-generated calendar of that employee is partially incorrect.
Do you think it should be filtered in another way?

Reply:
But this is not what you are checking right now. Anyways, it shouldn't be on the autogenerated calendar, but checking `calendar_ids` if there's a mix of both type.s